content/02.main-text.md

```diff
@@ -51,7 +48,7 @@ Conversations during review can reference other pull requests, issues, or author
 Reviewing batches of revisions that focus on a single theme is more efficient than independently discussing isolated comments and edits, and it helps maintain consistent content and tone across different authors and reviewers.
 Once all requested modifications are made, the manuscript maintainers, a subset of authors with elevated GitHub permissions, formally approve the pull request and merge the changes into the official version.
 `TODO: need a figure with a flowchart showing this process`
-The entire process can be orchestrated through GitHub with a web browser if a contributor does not want to use a git client on their own computer.
+The entire process can be orchestrated through GitHub with a web browser.
```

Comment:
There're some kinds of contributions or PR edits where this is not true. Perhaps we could make it more nuanced to indicate that "most types of contributions" or something along those lines?

Or is this sentence talking only about review and not creating / modifying the repo?

Reply:
Most of the content editing, including editing or creating new Markdown files, can be managed through the browser.  Is that the scope of the `process` we're discussing here?

Other operations, such as complex merges, certainly require a local git client.

Reply:
altered the scope lightly